### PR TITLE
Update workshop_call.md

### PR DESCRIPTION
**Pull request by shirin888**
Updated the workshop call for paper.

### CODE REVIEW

File: content/calls/workshop_call.md [hunk 1]
```diff
@@ -26,22 +24,21 @@ Workshop proposals should be no more than 3 pages long and should include the fo
...
-- List of accepted papers on the workshop website: 28 April 2023.
-- Workshop/Tutorial dates: 09-10 July 2023.
-
-Please direct all questions to the Workshop Program Chairs at <icaps2023-workshops@googlegroups.com>
-
+- Deadline for submission of workshop proposals: 12 January 2024.
+- Notification of workshop acceptance: 29 January 2024.
+- Workshop Call for Papers and materials for the website: 5 February 2024
+- List of accepted papers on the workshop website: 29 April 2024.
+- Workshop/Tutorial dates: 01-02 June 2024.
```
Comment (haz) [inline content/calls/workshop_call.md]: It'll actually be 2-3 (1st is DC)

File: content/calls/workshop_call.md [hunk 2]
```diff
@@ -26,22 +24,21 @@ Workshop proposals should be no more than 3 pages long and should include the fo
...
 
+Please direct all questions to the Workshop Program Chairs at icaps2024-workshops@googlegroups.com
```
Comment (haz) [inline content/calls/workshop_call.md:44]: We've set up the following alias for you already -- might be good to keep them uniform, unless you want the groups functionality?

workshops@icaps2024.com
- Biplav
- Shirin

**Review by haz — CHANGES_REQUESTED** [review]

File: content/calls/workshop_call.md (hunk 2 above)
Comment (shirin888) [reply content/calls/workshop_call.md:44]: I am inclined to keep the one we created.

File: content/calls/workshop_call.md (hunk 1 above)
Comment (shirin888) [reply content/calls/workshop_call.md]: Updated.

**Comment by haz** [comment]
@zlatanajanovic / @GloriaB ?

**Comment by shirin888** [comment]
I am not sure what the status of this pull request is, but it would be great if it went in, so the website is updated accordingly. I am already getting messages re the dates, etc. Thanks.

**Comment by zlatanajanovic** [comment]
I merged it now. Thanks!

**Pull request merged**